app/src/main/java/de/stephanlindauer/criticalmaps/fragments/MapFragment.java

```diff
@@ -223,6 +225,7 @@ public void onActivityCreated(final Bundle savedState) {
 
         mapView = MapViewUtils.createMapView(getActivity());
         mapContainer.addView(mapView);
+        zoomToLocation(defaultGeoPoint, DEFAULT_ZOOM_LEVEL);
```

Comment:
what was the initial location before that?

Reply:
Nothing, whole map was displayed

Reply:
https://user-images.githubusercontent.com/3222305/70018373-180aa680-15b0-11ea-8710-9e35866cd9a1.png

Reply:
hm. i am not a big fan of that because it makes the user think we are a berlin-focused app while we actually want to be used internationally. 
i do agree with @zutrinken that the current map with the split at antarctica looks weird but maybe a better fix would be to just zoom in a little closer to europe (it's kinda in the middle and still most of our userbase is located there) until that weird looking split disappears. 
i also think that a very global perspective is nice because if it is the last friday of the month the user can see how many people are using the app. 
in the end this will only be an issue for the first session, right? after that the app will remember your last view location, afair.
@headione @cbalster @lennet @mltbnz wdyt?

Reply:
@stephanlindauer good point, If all accept it, I'll change zoom level

Reply:
@stephanlindauer I agree with the global perspective, but showing streets and buildings seems more real imo. I see having the Berlin Mass as a default more as a nice hint to it's origin, but am also okay with a more zoomed out level. Just seeing the whole world feels more like flight radar.

Reply:
It's zoomLevel = 6, previous level was 12. Is it ok?

![image](https://user-images.githubusercontent.com/3222305/70372452-71a10700-1909-11ea-866f-a11442c4237b.png)

Reply:
what do you think about `DEFAULT_ZOOM_LEVEL = 3;`?
![image](https://user-images.githubusercontent.com/1323145/70374868-e4a18200-18f7-11ea-9609-af3b483330cd.png)

Reply:
@stephanlindauer ¯\_(ツ)_/¯ at least no odd Antarctica split

Reply:
![image](https://i.giphy.com/media/3ohzdIuqJoo8QdKlnW/giphy.webp)

:stuck_out_tongue_winking_eye: